docs/DART_LAB/matlab/private/update_inflate.m

```diff
@@ -1,10 +1,16 @@
 function [new_cov_inflate, new_cov_inflate_sd] = update_inflate(x_p, r_var, y_o, sigma_o_2, ss_inflate_base, lambda_mean, lambda_sd, ...
-                                                    inf_lower_bound, inf_upper_bound, gamma_corr, sd_lower_bound_in)
+                                                    inf_lower_bound, inf_upper_bound, gamma_corr, sd_lower_bound_in, ens_size, flavor)
 % Adaptive scheme to update the inflation both in space and time. 
-% Based on the algorithm in the following article:
+% Based on the algorithm in the following articles:
+%
+% [falvor: 2, Gaussian inflation pdf] 
 % Anderson, J. L., 2009: Spatially and temporally varying adaptive covariance 
 % inflation for ensemble filters. Tellus A, 61, 72-83. doi: 10.1111/j.1600-0870.2008.00361.x 
 %
+% [flavor: 5, I-Gamma inflation pdf] 
+% El Gharamti, M., 2018: Enhanced Adaptive Inflation Algorithm for Ensemble Filters. 
+% Monthly Weather Review, 146, 623-640. doi: 10.1175/MWR-D-17-0187.1
+%
```

Comment:
So ... the Matlab private/update_inflate.m  actually does the job of two Fortran routines (adaptive_inflate_mod.f90  linear_bayes() and enh_linear_bayes().

Granted, update_inflate.m  has fewer lines than the two Fortran routines, but I think the point is that we are trying to make it easy for the reader to go between the Matlab and Fortran worlds.   I'd like to split update_inflate.m into a
simple routine that calls one of two functions (to match the Fortran) ... those routines would be Matlab 'helper' functions in update_inflate.m
Thoughts?

Reply:
Good observation. My objective was to bring the functionality of flavor 5 just like what I did with flavor 2 around 3 years ago. See even when I brought in flavor 2 into DART_LAB, I didn't explicitly called the function linear_bayes() or adaptive_inflate_mode .. My intention was to make sure we are replicating the functionality but not mirroring the code. I think having the same parameter/variable names in enough for users to relate the fortran modules to the matlab functions.